web_widget_domain_editor_dialog/static/src/js/widget_domain_editor_dialog.js

```diff
@@ -0,0 +1,98 @@
+/* Copyright 2019 Tecnativa - David Vidal
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+odoo.define("web_widget_domain_editor_dialog.widget_domain_editor_dialog", function (require) {
```

Comment:
I'd call this module `web_wiget_domain_editor_dialog.DomainEditorDialog` and return that object directly.

Another option would be to have a single `web_widget_domain_editor_dialog` js module that performs all actions and returns the dict as it is doing now here.

Reply:
There you are :slightly_smiling_face: